matrix_client/api.py

```diff
@@ -741,8 +741,10 @@ def _send(self, method, path, content=None, query_params=None, headers=None,
             raise MatrixRequestError(
                 code=response.status_code, content=response.text
             )
-
-        return response.json()
+        try:
```

Comment:
It will be unclear why this is here without the context of the PR. More importantly, this doesn't feel very robust, what if the file to download is actually JSON?

I'd fix this by passing a new boolean to `_send`, not sure if this is the best way though.